src/sider/sykmeldinger/VurderBistandsbehov.tsx

```diff
@@ -29,21 +36,36 @@ const VurderBistandsbehov = ({ oppgave }: VurderBistandsbehovProps) => {
   const sykmelding = sykmeldinger.find(
     (sykmelding) => sykmelding.id === oppgave.referanseUuid
   );
+  const tiltakNAV = sykmelding?.arbeidsevne.tiltakNAV;
+  const tiltakAndre = sykmelding?.arbeidsevne.tiltakAndre;
+  const bistandsbehov = sykmelding?.meldingTilNav.navBoerTaTakISakenBegrunnelse;
   return !!sykmelding ? (
     <Panel className={"mb-4"}>
       <div className={"flex flex-row justify-between"}>
         <Heading size="medium" level="2">
           {texts.header}
         </Heading>
         <HelpText
-          title="Informasjon fra felt 8 i sykmeldingen"
+          title="Informasjon fra felter i sykmeldingen fra behandler"
           placement="left"
         >
           {texts.helptext}
         </HelpText>
       </div>
       <blockquote>
-        {sykmelding?.meldingTilNav.navBoerTaTakISakenBegrunnelse}
+        <BodyShort>
```

Comment:
Ikke kjempeviktig, men tror det er bedre med `{tiltakNav && <BodyShort>...teksten...</BodyShort>}` så slipper man å rendre en `BodyShort` med tom tekst hvis `tiltakNav` er null. Samme for de andre under.